### PR TITLE
Update Keaton Author page data and draft first article

### PR DESCRIPTION
**Pull request by keaton-taylor**
#### Purpose
This PR will add an author page to the site and includes my first article draft. 

#### Todos

- [ ] Edit article 
- [ ] Make sure all added code for author page makes sense

:coffee:

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -20,4 +20,14 @@ tjdraper:
...
-
+keatontaylor:
+  display_name: Keaton Taylor
+  title: Contributing Writer
+  twitter: keaton_taylor
+  github: keaton-taylor
+  web: www.keaton-taylor.com
+  email: keatontaylor@me.com
+  bio: >
+   Keaton is a designer and frontender working on editorial products at [Apartment Therapy Media](http://www.apartmenttherapy.com) from his home in sunny Southwest Missouri. Keaton loves his lovely wife and three children, craft beer, mediocre tv and Falltime.
```
Comment (smithtimmytim) [inline _data/authors.yml]: "Falltime" is technically not a thing. Which makes no sense considering that [every other season has one](http://english.stackexchange.com/questions/185221/why-is-there-no-autumntime-or-falltime). I say we keep it and start the "Falltime" trend.

File: _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: iOS
```
Comment (smithtimmytim) [inline _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown]: You can either come up with your own headline or what I do is "New <video> Policies for iOS | Webkit Blog"

File: _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: iOS 
+categories: link
+custom_type: link
+link_url: https://webkit.org/blog/6784/new-video-policies-for-ios/
+tags: [Video, Development]
+date: 2016-09-20 2:54
+---
```
Comment (smithtimmytim) [inline _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown]: Remember to add your author info to the YAML front matter.

File: _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Jer Noble for Webkit speaking on some new features in iOS 10:
+
+> `<video autoplay>` elements will now honor the `autoplay` attribute...
+
+This is an older piece of news but became relevant to me recently. For folks working on editorial and video platforms, it certainly has some interesting implications. Noble outlines some common-sense conditions that allow autoplay video in Mobile Safari. The conditions under which the `autoplay` attribute will be honored are pretty straightforward: 
+
```
Comment (smithtimmytim) [inline _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown]: I feel the paragraph works better like this:

> This is an older piece of news but became relevant to me recently. For folks working on editorial and video platforms, it certainly has some interesting implications. Noble outlines some common-sense conditions that allow autoplay video in Mobile Safari:

Comment (keaton-taylor) [inline _drafts/2016-09-20-iOS10-expands-autoplay-for-mobile.mdown]: 👍  agreed

**Comment by smithtimmytim** [comment]
@keaton-taylor Changes look good. Last thing to do is put it in the `_posts` folder.

**Comment by keaton-taylor** [comment]
I really like this new format on GH.

File: _data/authors.yml [hunk 5]
```diff
@@ -20,4 +20,14 @@ tjdraper:
     TJ is a independent web developer and owner of [BuzzingPixel, LLC](https://buzzingpixel.com/) in Spring Hill, TN. He is also a veteran podcaster — having survived 150 episodes as host of [The MovieByte Podcast](https://moviebyte.com/mbpodcast). Now TJ hosts a new show about appreciating cinema called [Retake](https://nightowl.fm/retake) on the podcast network he launched with [Joe Darnell](https://nightowl.fm/people/joedarnell), [Night Owl](https://nightowl.fm/).
   author_path: tjdraper
   gravatar: 979fa599e691b040164ca31d848a9cf9
-
+keatontaylor:
+  display_name: Keaton Taylor
+  title: Contributing Writer
+  twitter: keaton_taylor
+  github: keaton-taylor
+  web: www.keaton-taylor.com
```
Comment (smithtimmytim) [inline _data/authors.yml]: Shoot. Looks like this doesn't work. You'll have to put `http://keaton-taylor.com`

Comment (keaton-taylor) [inline _data/authors.yml]: np

**Pull request merged**